src/modules/music/mod.rs

```diff
@@ -183,18 +183,18 @@ impl Module<Button> for MusicModule {
 
         icon_pause.label().set_justify(self.layout.justify.into());
 
-        let label = Label::builder()
-            .use_markup(true)
-            .justify(self.layout.justify.into())
-            .build();
-
-        if let Some(truncate) = self.truncate {
-            label.truncate(truncate);
-        }
+        let label = OverflowLabel::new(
+            Label::builder()
+                .use_markup(true)
+                .justify(self.layout.justify.into())
+                .build(),
+            self.truncate,
+            self.marquee.clone(),
+        );
```

Comment:
I'd prefer the initial label separated here, just to make it a bit more readable:

```suggestion
        let label = Label::builder()
          .use_markup(true)
          .justify(self.layout.justify.into())
          .build();
        
        let label = OverflowLabel::new(
            label,
            self.truncate,
            self.marquee.clone(),
        );
```